ambry-frontend/src/main/java/com/github/ambry/frontend/s3/S3MultipartUploadPartHandler.java

```diff
@@ -104,10 +101,10 @@ void handle(RestRequest restRequest, RestResponseChannel restResponseChannel,
     String uploadId = RestUtils.getHeader(restRequest.getArgs(), UPLOAD_ID_QUERY_PARAM, true);
     restRequest.setArg(S3_CHUNK_UPLOAD, true);
     restRequest.setArg(SESSION, uploadId);
-    // TODO [S3] : set part ttl value
```

Comment:
@weisong44 @asinghai 

I'm thinking, 
1. For part upload, we always use 29 days or 30 days ttl. We don't let user to control it. So if the part upload fails or user doesn't complete the multipart upload, these blobs will expire in one month.
2. In the CompleteMultiPart callchain, we set ttl to permanent.

We may have some discussion about it.

Reply:
Thanks! Based on our offline discussion, I will make the following changes
1. Use `chunkUploadInitialChunkTtlSecs` for upload
2. Use `permanentNamedBlobInitialPutTtl` during stitching